sklearn/multioutput.py

```diff
@@ -648,6 +648,51 @@ def _log_message(self, *, estimator_idx, n_estimators, processing_msg):
             return None
         return f"({estimator_idx} of {n_estimators}) {processing_msg}"
 
+    def _get_chain_method(self):
+        # Different chain methods allowed in `ClassifierChain` only
+        try:
+            chain_method = self.chain_method_
+        except AttributeError:
+            chain_method = "predict"
+        return chain_method
+
+    def _get_Y_output(self, X, *, output_method="predict"):
+        """Get predictions for each model in the chain."""
+        check_is_fitted(self)
+        X = self._validate_data(X, accept_sparse=True, reset=False)
+        Y_output_chain = np.zeros((X.shape[0], len(self.estimators_)))
+        Y_feature_chain = np.zeros((X.shape[0], len(self.estimators_)))
+
+        chain_method = self._get_chain_method()
+        # proba methods produce 2d output (decision_function 1d for binary targets)
```

Comment:
For binary classifier, we don't want to have the 2 columns because they are correlated. But the `_get_response_values` will also make this work for you.